M/MozillaCACerts/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MozillaCACerts"
+version = v"1.0.0"
```

Comment:
What about using the date as version?

Reply:
I was thinking about that -- `2020.01.01`? Or will it be `2020.1.1`? Will these be sorted correctly?

Reply:
It'll be a `VersionNumber`, so it should be sorted as expected:
```julia
julia> v"2020.02.01"
v"2020.2.1"

julia> v"2020.02.01" < v"2020.11.01"
true
```